DedicatedServer/Scripts/Libs/Zrx/ModeLibs/FlagRush/UI/Modules/Respawn.Script.txt

```diff
@@ -0,0 +1,48 @@
+//#RequireContext CMlScriptIngame
+
+Text GetManialink() {
+	return """
+	<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
+	<manialink version="3" name="FlagRush_RespawnUI">
+		<frame pos="-40 -50" id="rs-frame">
+			<label pos="40 -5" z-index="0" size="80 6" text="" halign="center" style="TextCardSmall" textsize="3" valign="center2" id="rs-label"/>
+			<gauge pos="0 0" z-index="0" size="80 10" style="BgCard" ratio="1" id="rs-gauge"/>
```

Comment:
This gauge has a set defult color which can not realy be changed. It can partially be changed with modulate color or whatever, but the default blue color will remain partially, polluting the pontentially modified color.

The base color of this style is never used anywhere else. I would like to keep UI consistency as far as possible. Is there another style we could use? Maybe just use EnergyBar (same aus FlagGauge) without it's background and add a custom one.

Note about EnergyBar: For somereason the ratio of energy bar seems to end at 0.2 on the lower end. I don't know if that's also the case for some other gauges.
See `UI/Modules/Markers.Script.txt`, line 93:
```
declare AdjustedRatio = ML::Clamp(0.2 + Ratio * 0.8, 0.2, 1.0);	// Somehow the gauge ends at 0.2? Also clamp, as we had an exception before
```